tests/elements-regression/scripts/run-on-linux.js

```diff
@@ -20,10 +20,10 @@ async function run( args ) {
 		`--volume ${ workingDir }:/work`,
 		'--workdir /work/',
 		'--interactive',
-		'--tty',
+		process.env.CI ? '' : '--tty',
```

Comment:
why it runs on the CI?

Reply:
It doesn't run in the CI, but I want it to be able to run in the CI since from some version of Playwright, there is a 1px height difference if this is run locally (even from a Docker container, see: https://github.com/microsoft/playwright/issues/20366 ). So I want to be able to run it in the CI so I could get the correctly generated screenshots.